app/helper.js

```diff
@@ -267,13 +299,17 @@ class Helper {
       const guildId = message && message.guild && message.guild.id ?
         message.guild.id :
         Array.from(this.client.guilds)
-          .find(guild => guild[1].members.has(message.author.id))[0],
+          .find(guild => guild[1].members.has(message.author.id))[0];
 
-        botChannelString = !!guildId ?
+        let botChannelString = !!guildId ?
```

Comment:
What was the rationale behind this change?

Reply:
It didn't make sense to me, but it wasn't working unless I included let. It was a change I made just to test and then... it worked.

Reply:
It looks like you changed line 302 from a `,` to a `;`, which negates the chaining of the variable definitions.

This is why it would have been giving an error, but is an unnecessary change.